ELLI/src/app/features/signup/signup.component.html

```diff
@@ -1,7 +1,7 @@
 <section class="login-container">
     <div class="container">
-        <h2 class="center">Signup</h2>
-        <p class="login-link" (click)="showLogin()">Login</p>
+        <h2 class="center">Login</h2>
+        <p class="login-link" (click)="showSignup()">Sign Up</p>
```

Comment:
needs to be changed to showLogin() this crashes the program since showSignup() is not defined in the .ts file

Reply:
following will take notes regarding that .